packages/prop-controllers/src/link.ts

```diff
@@ -32,21 +33,45 @@ export type Link =
   | CallPhoneLink
   | ScrollToElementLink
 
-export type LinkValue = Link
+export type LinkPropControllerDataV0 = Link
+
+export const LinkPropControllerDataV1Type = 'prop-controllers::link::v1'
+
+export type LinkPropControllerDataV1 = {
+  [ControlDataTypeKey]: typeof LinkPropControllerDataV1Type
+  value: Link
+}
+
+export type LinkPropControllerData =
+  | LinkPropControllerDataV0
+  | LinkPropControllerDataV1
 
 export type LinkOptions = Options<{
-  preset?: LinkValue
+  preset?: LinkPropControllerData
   label?: string
   defaultValue?: Link
   options?: { value: Link['type']; label: string }[]
   hidden?: boolean
 }>
 
-export type LinkDescriptor<_T = LinkValue> = {
+export type LinkDescriptor<_T = LinkPropControllerValue> = {
```

Comment:
@migueloller this should use the value and not the data right?

Reply:
I think that the generics we use here are the descriptor data. Mapping it to a value is done separately.

Reply:
Ok updated to `LinkPropControllerData`